EnglishFilter/sections/specific.txt

```diff
@@ -322,7 +322,7 @@ freetutorials-us.com##.ultra-archive-grid
 allinonedownloadzz.site##.tdi_114
 allinonedownloadzz.site##a[href^="https://vgw35hwr4w6x.com/"]
 3dsportal.net###rightside > .block > .btl
-skidrowcodexgame.com,skidrowcodexreloaded.com##.footer-sticky
```

Comment:
It seems this rule is still relevant, I added` skidrowcodexreloaded.com#@#.footer-sticky` and the ad is still there:
![relevant](https://user-images.githubusercontent.com/18536328/134789262-1aa6e372-e5b3-484b-a215-959ac1ef05ec.png)